src/command.c

```diff
@@ -1170,3 +1170,57 @@ cmd_replaygain(int argc, char **argv, struct mpd_connection *connection)
 
 	return 0;
 }
+
+int
+cmd_partitionlist(int argc, char **argv, struct mpd_connection *conn)
+{
+    if (!mpd_send_listpartitions(conn)) {
```

Comment:
Please use tabs for indentation, like the rest of the code base.

Reply:
Fixed.